tests/test_unittest/test_gtc/test_passes/test_oir_optimizations/test_horizontal_execution_merging.py

```diff
@@ -35,6 +37,22 @@
 )
 
 
+def test_horiz_exec_extents():
+    stencil = StencilFactory(
+        vertical_loops__0__sections__0__horizontal_executions=[
+            HorizontalExecutionFactory(
+                body=[AssignStmtFactory(left__name="tmp", right__name="input")]
+            ),
```

Comment:
Just having `HorizontalExecutionFactory(body__0__left__name="tmp")` should also work. You do not have to specify the `right__name`, a unique value is generated automatically (and omission helps the reader to understand that its value is irrelevant).